frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -629,21 +627,25 @@ export default ({
       this.messages.splice(index, 1)
       this.handleSendMessage(message.text, message.attachments, message.replyingMessage)
     },
-    replyMessage (message) {
-      const { text, hash, type } = message
+    replyToMessage (message) {
+      const { text, hash, type, attachments } = message
+      const isTypeInteractive = type === MESSAGE_TYPES.INTERACTIVE
 
-      if (type === MESSAGE_TYPES.INTERACTIVE) {
+      if (isTypeInteractive) {
         const proposal = message.proposal
 
         this.ephemeral.replyingMessage = {
-          text: interactiveMessage(proposal, { from: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${proposal.creatorID}` }),
-          hash
+          hash, text: interactiveMessage(proposal, { from: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${proposal.creatorID}` })
         }
-        this.ephemeral.replyingTo = L('Proposal notification')
+      } else if (!text && attachments.length) {
+        this.ephemeral.replyingMessage = { hash, text: attachments[0].name }
       } else {
-        this.ephemeral.replyingMessage = { text, hash }
-        this.ephemeral.replyingTo = this.who(message)
+        this.ephemeral.replyingMessage = {
+          hash, text: stripMarkdownSyntax(text, CHATROOM_REPLYING_MESSAGE_LIMITS_IN_CHARS)
+        }
```

Comment:
Another thought though is: you could also display the markdown, just not render it. What do you think?

Reply:
That will simplify things. I can make this update.